src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -188,7 +188,7 @@ bool MulticopterLandDetector::_get_ground_contact_state()
 	}
 
 	if (lpos_available && _vehicle_local_position.dist_bottom_valid) {
```

Comment:
```suggestion
	if (lpos_available && _vehicle_local_position.dist_bottom_valid && _param_lndmc_alt_gnd_effect.get() > 0) {
```
Maybe just to be sure if there's a misbehaving distance sensor driver that publishes a negative value.

Reply:
okay makes sense, added it